effect/src/main/scala/org/typelevel/catbird/util/effect/package.scala

```diff
@@ -16,6 +16,8 @@ package object effect extends FutureInstances with RerunnableInstances {
       case Return(a)  => k(Right[Throwable, A](a))
       case Throw(err) => k(Left[Throwable, A](err))
     }
+
+    ()
```

Comment:
Explicitly return unit to indicate that we're ok discarding the `Future` returned by `fa.respond`